src/rules/prefer-lowercase-title.ts

```diff
@@ -54,7 +54,8 @@ export default createEslintRule<[
     },
     fixable: 'code',
     messages: {
-      lowerCaseTitle: '`{{ method }}`s should begin with lowercase'
+      lowerCaseTitle: '`{{ method }}`s should begin with lowercase',
+      fullyLowerCaseTitle: '`{{ method }}`s should be fully lowercase'
```

Comment:
how about we improve the wording and say 
```suggestion
      fullyLowerCaseTitle: '`{{ method }}`s should be lowercase'
```